test/types/community/collection/updateX.test-d.ts

```diff
@@ -221,7 +221,10 @@ expectError<UpdateFilter<TestModel>>({
   $set: { 'subInterfaceField.nestedObject': { a: 1, b: '2' } }
 });
 expectError(buildUpdateFilter({ $set: { 'subInterfaceField.field2': 2 } }));
-expectError(buildUpdateFilter({ $set: { 'unknown.field': null } }));
+
+// NODE-3875 introduced intersection with Document to the MatchKeysAndValues so this no longer errors
+// expectError(buildUpdateFilter({ $set: { 'unknown.field': null } }));
```

Comment:
Instead of commenting out the test, should we instead `expectAssignable`?

Reply:
Yea, agreed, fixed!